test/blackbox-tests/test-cases/install-with-var/run.t

```diff
@@ -0,0 +1,365 @@
+`dune install` should handle destination directories that don't exist
+
+  $ dune build @install
+  $ dune install --prefix install --libdir lib
+  Installing install/lib/foo/META
+  Installing install/lib/foo/foo$ext_lib
+  Installing install/lib/foo/foo.cma
+  Installing install/lib/foo/foo.cmi
+  Installing install/lib/foo/foo.cmt
+  Installing install/lib/foo/foo.cmx
+  Installing install/lib/foo/foo.cmxa
+  Installing install/lib/foo/foo.cmxs
+  Installing install/lib/foo/foo.dune
+  Installing install/lib/foo/foo.ml
+  Installing install/lib/foo/opam
+  Installing install/bin/exec
+  Installing install/man/man1/a-man-page.default.1
+  Installing install/man/man3/another-man-page.3
+
+If prefix is passed, the default for libdir is `$prefix/lib`:
```

Comment:
The rest of this file shouldn't be necessary, the first test is enough to check that variable expansion works correctly.